app/styles/app.scss

```diff
@@ -0,0 +1,87 @@
+$gutter: 1rem;
+
+body {
+  margin: 0;
+}
+
+ul {
+  padding-left: 0;
+  list-style: none;
+}
+
+figure {
+  list-style: none;
+}
+
+.app {
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+
+}
+
+.app-nav {
+  width: 100%;
+  background: #e2f7fa;
+}
+
+.container {
+  max-width: 65rem;
+  width: 100%;
+  margin: 0 auto;
+  display: flex;
+}
+
+.app-main {
+  background: #000;
+  flex-grow: 1;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  overflow: auto;
+
+}
+
+.brewery-list {
+  display: flex;
```

Comment:
To make the list items line up with things that aren't in the row, use a negative margin left and right to make up for the first item's margin.

Also use either justify content or flex-grow to make the rows respond better at smaller screen sizes